Header 								header
string                              model
bool                             loaded
bool                             active
